README.md

```diff
@@ -8,6 +8,10 @@ Lotus is an implementation of the Filecoin Distributed Storage Network. For more
 
 For instructions on how to build lotus from source, please visit [https://docs.lotu.sh](https://docs.lotu.sh) or read the source [here](https://github.com/filecoin-project/lotus/tree/master/documentation).
 
+## Reporting a Vulnerability
+
+Please send an email to security@filecoin.org. See our [security policy](SECURITY.md) for more details.
```

Comment:
Is that gonna link to the SECURITY page correctly (i'm not sure)?

Reply:
yes, if not fully qualified it defaults to the root of the repo

Reply:
What is the usual response time for reporting a bug via that email?  Or is there a better place to post?